pkgs/development/python-modules/pyimpfuzzy/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pkgs
+, pefile
+}:
+buildPythonPackage rec {
+  pname = "pyimpfuzzy";
+  version = "0.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "da9796df302db4b04a197128637f84988f1882f1e08fdd69bbf9fdc6cfbaf349";
+  };
+
+  buildInputs = [
+    pkgs.ssdeep
+  ];
+
+  propagatedBuildInputs = [
+    pefile
+  ];
+
+  meta = with lib; {
+    description = "A Python module which calculates and compares the impfuzzy (import fuzzy hashing)";
+    homepage = "https://github.com/JPCERTCC/impfuzzy";
+    license = licenses.gpl2; # unclear about variant
```

Comment:
~~Where is that mentioned? Their repo does not list any license which would mean it is unfree.~~

What so hard about using the default license name? ``the GNU General Public License version 2``

Thats gpl2Only.

Reply:
Oh, does GPL2 default to `gpl2Only`? Good to know.